logic/src/commonTest/kotlin/com/wire/kalium/logic/data/message/receipt/ReceiptRepositoryTest.kt

```diff
@@ -57,8 +57,45 @@ class ReceiptRepositoryTest {
         }
     }
 
+    @OptIn(ExperimentalCoroutinesApi::class)
+    @Test
+    fun givenMessageReadReceiptsWerePersisted_whenObservingMessageReceipts_thenShouldReturnReceiptsPreviouslyStoredAndOrdered() = runTest {
```

Comment:
Please rename the test so we know what's the criteria for ordering.

I guess user name alphabetically in this case.